commands.js

```diff
@@ -22,6 +22,8 @@ const TEST_COMMAND = {
   name: 'test',
   description: 'Basic command',
   type: 1,
+  integration_types: [0, 1],
```

Comment:
I am not 100% sure we need these.  I believe we default to all interaction contexts (well, at least according to https://discord.com/developers/docs/interactions/application-commands#interaction-contexts we do).  Integration types for the command should be set based on which installation types are enabled on the app (again, we should verify this).  The intent of those late changes to User Apps was so these fields wouldn't be needed in the default case.